frontend/src/streamlist/StreamsTable.jsx

```diff
@@ -83,7 +89,14 @@ export default class StreamsTable extends React.Component {
   }
 
   getColumnInfo() {
-    return this.columnInfo;
+    const { isSimple } = this.props;
+    let result;
+    if (isSimple) {
+      result = this.simpleColumnInfo;
+    } else {
+      result = this.columnInfo;
+    }
```

Comment:
can't you just put `return this.simpleColumnInfo` and `return this.columnInfo` instead of using `result` here?

Reply:
No because the linter complains and removes my else block and I want to make the linter happy.

Reply:
right. disgusting (of the linter to do that). have a nice day

Reply:
Yeah, I thought the else statement getting erased by the linter was going to cause issues but I am just forgetting how coding works so I will be fixing.